eras/shelley/impl/cardano-ledger-shelley.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-shelley
-version:            1.8.0.0
+version:            1.9.0
```

Full version must be specified at all times, even in the changelog. This is necessary for consistency, because cabal doesn't treat those equally `1.9.0` and `1.9.0.0` 
```suggestion
version:            1.9.0.0
```